py/mpconfig.h

```diff
@@ -490,6 +490,11 @@
 #define MICROPY_LONGINT_IMPL (MICROPY_LONGINT_IMPL_NONE)
 #endif
 
+// Support for calls to buildin.pow() with 3 integer arguments
+#ifndef MICROPY_INT_POW3
+#define MICROPY_INT_POW3 (1)
```

Comment:
OK, I'm fine with leaving this off by default. It's fairly easy to implement pow3 in a few lines of Python anyway. Perhaps we could put the minimal Python implementation into the text of the NotImplemented exception :-)